README.md

```diff
@@ -32,7 +32,7 @@
   <td>2018-04-01</td>
 </tr>
 <tr>
-  <td>N/A</td>
+  <td>End-of-Life</td>
```

Comment:
Since this is in a `LTS Status` column, `End-of-Life` may be misinterpreted as suggesting this was previously LTS (it never was).